### PR TITLE
feat(apps/gtodo-api): add title for todos

### PR DESCRIPTION
**Pull request by amir25n**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/gtodo-api/src/index.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ let requestCounter = 0;
 const routerLogger = logger.scope('router');
 
 server.server.on('request', (request, response) => {
-  const label = `(${String(++requestCounter).padStart(5, '·')}) ${
+  const label = `(${String(++requestCounter).padStart(5,'·')}) ${
```
Comment (MM25Zamanian) [inline apps/gtodo-api/src/index.ts:25]: چرا کد رو زشت میکنی

File: apps/gtodo-api/src/libs/app.ts [hunk 2]
```diff
@@ -39,6 +39,7 @@ export const appRouter = router({
   edit: publicProcedure
       .input(
           todoInputValidation.required({ id: true }).partial({
+            title : true,
```
Comment (MM25Zamanian) [inline apps/gtodo-api/src/libs/app.ts:42]: پریتیر بزن

File: apps/gtodo-api/src/libs/db.ts [hunk 3]
```diff
@@ -9,6 +9,7 @@ import config from '../config';
 
 export interface TodoInterface {
   status: boolean;
+  title : string;
```
Comment (MM25Zamanian) [inline apps/gtodo-api/src/libs/db.ts:12]: پرییتیر

File: apps/gtodo-api/src/libs/db.ts [hunk 4]
```diff
@@ -19,6 +20,7 @@ export const todoInputValidation = z
     .object({
       status: z.boolean(),
       message: z.string(),
+      title : z.string(),
```
Comment (MM25Zamanian) [inline apps/gtodo-api/src/libs/db.ts:23]: پریییییییتییییییر

**Review by MM25Zamanian — CHANGES_REQUESTED** [review]

File: apps/gtodo-api/src/libs/db.ts (hunk 4 above)
Comment (amir25n) [reply apps/gtodo-api/src/libs/db.ts]: jesus christ 🤡

**Pull request closed without merging**